### ISSUE TITLE
Payload doesn't match '^0000[0-9A-F]{2}(00|10|20)': 01000010

### ISSUE THREAD

**Issue by keesma** [zxdavb/ramses_cc#156]
I found the following line in the home assistant log of version 0.31.7:

`2024-02-05 08:02:41.237 WARNING (MainThread) [ramses_tx.message]  I --- 30:069468 --:------ 30:069468 3110 004 01000010 < Payload doesn't match '^0000[0-9A-F]{2}(00|10|20)': 01000010
`

My HVAC information is not updated (anymore?)

Some other parts of the homeasistant log:
```
2024-02-04 23:33:12.372 INFO (MainThread) [custom_components.ramses_cc.binary_sensor] Found 30:069468 (FAN): bypass_position

2024-02-04 23:33:12.372 INFO (MainThread) [custom_components.ramses_cc.sensor] Found 30:069468 (FAN): air_quality
2024-02-04 23:33:12.372 INFO (MainThread) [custom_components.ramses_cc.sensor] Found 30:069468 (FAN): co2_level
2024-02-04 23:33:12.372 INFO (MainThread) [custom_components.ramses_cc.sensor] Found 30:069468 (FAN): exhaust_fan_speed
2024-02-04 23:33:12.372 INFO (MainThread) [custom_components.ramses_cc.sensor] Found 30:069468 (FAN): exhaust_flow
2024-02-04 23:33:12.372 INFO (MainThread) [custom_components.ramses_cc.sensor] Found 30:069468 (FAN): exhaust_temp
2024-02-04 23:33:12.372 INFO (MainThread) [custom_components.ramses_cc.sensor] Found 30:069468 (FAN): fan_info
2024-02-04 23:33:12.372 INFO (MainThread) [custom_components.ramses_cc.sensor] Found 30:069468 (FAN): filter_remaining
2024-02-04 23:33:12.372 INFO (MainThread) [custom_components.ramses_cc.sensor] Found 30:069468 (FAN): indoor_humidity
2024-02-04 23:33:12.372 INFO (MainThread) [custom_components.ramses_cc.sensor] Found 30:069468 (FAN): indoor_temp
2024-02-04 23:33:12.372 INFO (MainThread) [custom_components.ramses_cc.sensor] Found 30:069468 (FAN): outdoor_humidity
2024-02-04 23:33:12.372 INFO (MainThread) [custom_components.ramses_cc.sensor] Found 30:069468 (FAN): outdoor_temp
2024-02-04 23:33:12.372 INFO (MainThread) [custom_components.ramses_cc.sensor] Found 30:069468 (FAN): post_heat
2024-02-04 23:33:12.372 INFO (MainThread) [custom_components.ramses_cc.sensor] Found 30:069468 (FAN): pre_heat
2024-02-04 23:33:12.372 INFO (MainThread) [custom_components.ramses_cc.sensor] Found 30:069468 (FAN): remaining_mins
2024-02-04 23:33:12.372 INFO (MainThread) [custom_components.ramses_cc.sensor] Found 30:069468 (FAN): supply_fan_speed
2024-02-04 23:33:12.372 INFO (MainThread) [custom_components.ramses_cc.sensor] Found 30:069468 (FAN): supply_flow
2024-02-04 23:33:12.372 INFO (MainThread) [custom_components.ramses_cc.sensor] Found 30:069468 (FAN): supply_temp

2024-02-04 23:33:12.372 INFO (MainThread) [custom_components.ramses_cc.climate] Found HVAC 30:069468 (FAN)
```

Config
```
ramses_cc:
  serial_port: /dev/serial/by-id/usb-SparkFun_evofw3_atmega32u4-if00

  packet_log:
    file_name: packet.log
    rotate_backups: 28

  ramses_rf:
    enforce_known_list: true
    enable_eavesdrop: false

  restore_cache: false #  turn on later, is off temporarily
  
  01:026398:  {} # looks like a new number, added {}

  known_list:
    18:204301: { class: HGI }  # ? Active gateway (USB or BDR91, Evohome touch)
    01:026398: { alias: "Controller" }
    04:014773: { alias: "N" }     # TRV n
    04:145139: { alias: "S" }  # TRV s
    04:145175: { alias: "W" }   # TRV w
    04:014793: { alias: "C" }       # TRV c
    04:145159: { alias: "A" }        # TRV a
    05:197404: { alias: "W" }   # Temperature sensor
    01:145038:               # evotouch?
    13:019795: { alias: "CV relay" }       # BDR91
    13:198915: { alias: "Zone valve" }     # BDR91, zone valve
    22:197404:                          # sensor
    30:069468: {class: FAN}  # suggestion from David
```

**Comment by zxdavb** [comment]
This is a warning - it should not be causing your problem:
 - My HVAC information is not updated (anymore?)

**Comment by zxdavb** [comment]
Has a look in your log files, or send me a packet log.

**Comment by keesma** [comment]
I will share the packet log with you.

**Comment by zxdavb** [comment]
> I will share the packet log with you.

I can't see where you sent it.

**Comment by zxdavb** [comment]
Was it a none-off error - or do you get a lot of them?

**Comment by zxdavb** [comment]
... and I have all the info I need in #159

**Comment by zxdavb** [comment]
will be fixed in **0.31.10** (and also **0.41.10**).
